geonode_dominode/geonode_dominode/templates/groups/group_detail_override.html

```diff
@@ -55,15 +55,15 @@ <h2 class="page-title">{{ object.title|default:object.slug }}</h2>
       {% if can_view %}
       <li class="list-group-item"><a href="{% url "group_activity" object.slug %}">{% trans "Group Activities" %}</a></li>
       {% endif %}
-      {% if perms.groups.can_sync_geoserver %}
-      <li class="list-group-item">
-        <form action="{% url "sync_geoserver" %}" method=post>
-          {% csrf_token %}
-          <input type="hidden" name="group-slug" value="{{ object.slug }}" />
-          <input type="hidden" name="redirect" value="{{ request.path }}" />
-          <input type="submit" class="btn btn-primary" value="{% trans "Sync GeoServer" %}" />
-        </form>
-      </li>
+      {% if is_member and perms.groups.can_sync_geoserver %}
```

Comment:
Now I am unsure `perms.groups` is what we really want. Because based on the issue that you raised, that means we want to check the perms of the group that is currently being viewed?

Reply:
Check the attached PR in your Repo, I made a small diff.